### ISSUE TITLE
Delete treasury report when requested force-regenerate

### ISSUE THREAD

**Issue by as1729** [usdigitalresponse/cpf-reporter#510]

**Comment by ClaireValdivia** [comment]
This should address issue where projects in overwritten or invalidated files are included in the treasury report

**Comment by nowei** [comment]
Just making sure, but there's no force-regenerate button today, right?

**Comment by as1729** [comment]
@nowei that is correct there is no button today to force-regenerate. This ticket will also require a front-end button to kick-off force regenerate. For now that button can be added to `/treasury-report-developer-tools` URL.

**Comment by as1729** [comment]
@nowei I created #543 so you can attach the front-end button changes PR to the new ticket.
